src/Features/CSharp/Portable/CodeRefactorings/ConvertLocalFunctionToMethod/CSharpConvertLocalFunctionToMethodCodeRefactoringProvider.cs

```diff
@@ -52,8 +52,11 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
 
             var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
-            context.RegisterRefactoring(new MyCodeAction(CSharpFeaturesResources.Convert_to_method,
-                c => UpdateDocumentAsync(root, document, parentBlock, localFunction, c)));
+            context.RegisterRefactoring(
+                new MyCodeAction(CSharpFeaturesResources.Convert_to_method,
+                            c => UpdateDocumentAsync(root, document, parentBlock, localFunction, c)),
+                localFunction.Span
+                );
```

Comment:
ehhh.... :)

Reply:
Yeah, I need to figure out better wrapping. That said, they all look terrible.

Reply:
```
context.RegisterRefactoring(
  new MyCodeAction(
    CSharpFeaturesResources.Convert_to_method,
    c => UpdateDocumentAsync(root, document, parentBlock, localFunction, c)),
  localFunction.Span);
```